### ISSUE TITLE
translation factors, various

### ISSUE THREAD

**Issue by ValWood** [geneontology/go-ontology#29089]
I'm trying to build a GO-CAM of mitochondrial translation , and I see many issues with translation factors.

<img width="1185" alt="Screenshot 2024-10-23 at 14 18 34" src="https://github.com/user-attachments/assets/0a0d0752-8b01-46b0-bd93-02946b6cf8d7">

- [x] 1. translation initiation factor 
is under "translation initiation" but is also under regulation fo translation".
This seems incorrect, the initiation factors are part_of translation?

FIXED

- [ ] 2. We don't have a way to describe the individual activities of "translation initiation factors"
 These include:

Inhibit the recruitment of the ribosome large subunit to the small subunit (via mtIF3) (until the initiation complex is properly assembled)
Recruit the small ribosomal subunit to the initiator tRNA (Formation of the Pre-Initiation Complex (PIC)) involving mtIF2 and fMet-tRNAi).
Recruit small ribosomal subunit to the mRNA (no need for cap recognition or extensive mRNA recruitment).
Recruit initiator tRNA to the small ribosomal subunit (via mtIF2).
Start codon recognition (via mtIF2 and GTP hydrolysis).
Recruit the large subunit to form the complete initiation complex.

(for all initiation and a few extra activities for eukaryotic)

I think we can model this using existing activities. I will request new terms as required when I build the model

- [ ]  3. The description of "translation iniitation" factor is vague
Defined Functions in the initiation of ribosome-mediated translation of mRNA into a polypeptide.

but there are factors that act upstream of these iniitation  steps in 'pre-initiation'  (PPR proteins that act generally to recruit mRNAs, and to recruit TIFs to the ribosome are not themselves considered to be TIFs but fit this definition).

I am leaving this for now because the definition needs to fit mitochondrial and cytosolic and bacterial translation. I don't  know enough about the difference right now

**Comment by ValWood** [comment]
4. mRNA regulatory element binding translation repressor activity
should be is_a 
 translation factor activity, RNA binding

**Comment by ValWood** [comment]
5. translation initiation factor activity
is_a translation factor activity RNA binding but this is not always true

**Comment by ValWood** [comment]
6. translation activator activity  is a subclass of “translation regulator activity, nucleic acid binding
but some translation factors don’t bind  mRNA directly

**Comment by ValWood** [comment]
7. the positioning of inititation, elongation and termination factors under 
GO:0008135  translation factor activity, RNA binding
is problematic because not all of these factors bind to RNA it seems, for example "mtEFG1: This factor is involved in the translocation of the ribosome along the mRNA after peptide bond formation, similar to EF-G in prokaryotic translation. It interacts with the ribosome but does not bind mRNA directly" there appear to be a number of translation factors that  son't bind to RNA (also this doesn't seem to be an ideal way to classify these activities)

**Comment by ValWood** [comment]
Step 1, obsolete the terms with "nucleic acid binding"
Step 2 separate "translation factors from translation regulators GO:0008135  
translation factor activity, RNA binding
Step 3 make the RNA binding terms children of the more general terms

**Comment by ValWood** [comment]
The problems here are big.

Translation regulator activity is defined:
Any molecular function involved in the initiation, activation, perpetuation, repression or termination of polypeptide synthesis at the ribosome.

So this includes all translation factors, involved in translation and has the parentage:

<img width="573" alt="Screenshot 2024-12-02 at 14 35 03" src="https://github.com/user-attachments/assets/95b16b87-ca56-4d2a-9d31-6cd0ae1050e7">

This means that ALL translation factors, become annotated to "regulation of translation" even if they are actually part_of translation. This is a perpetual problem with  making MF term "translation factor"  . I'm not sure how to fix this.

**Comment by ValWood** [comment]
We could have a term "translation factor" defined as the translation regulator is defined now:

i.e Any molecular function **involved in** the initiation, activation, perpetuation, repression or termination of polypeptide synthesis at the ribosome.

and remove the terms for "translation regulator"

**Comment by ValWood** [comment]
Here is a redical proposal.
We obsolete all of the translation factor/translation regulator terms.
We should be annotating to the activities,a nd if we don't know tham, just the process.

So for termination factor   PomBase has 10 annotations
8 have an activity in addition. to translation termination factor
erf1 GTPase regulator activity
sup35 GTPase activity
hsb1 GTPase activity
dom35 GTPase regulator activity
pth3 aminoacyl-tRNA hydrolase activity
pth4 aminoacyl-tRNA hydrolase activity
mrf1 ? 
rrf1 ? this might be ribosome disassembly
tif51 ? possibly a ribosome RNA adaptor?
tif512 ? possibly a ribosome RNA adaptor?

so the question  (which has been raised before) is what do we mean by "translation factors" other than "an entity involved in translation". These should really be synonyms of the related process rather than trying to collect a bag of unrelated functions.

Then we would only need to deal with "translation regulator" which  could probably be similarly addressed?

**Comment by ValWood** [comment]
Some of the translation regulators probably make sense but  need better labels

GO:1903231  mRNA base-pairing translational repressor activity
Definition ([GO:1903231 GONUTS page](http://gowiki.tamu.edu/wiki/index.php/Category:GO:1903231))
A translation repressor activity that acts by base-pairing with an mRNA. The binding can result in targeting the mRNA for degradation or interfering with mRNA translation, hence resulting in posttranscriptional gene silencing. PMID:18197166 PMID:20533884

Comments
This term is intended for microRNAs that act by base-pairing with a target mRNA.

reading this term label, I would not know it was intended for microRNAs

**Comment by pgaudet** [comment]
Please feel free to suggest a better label. 
@RLovering @hattrill @gantonazzo and I didn't love it either! but we couldn't think of anything better at the time.

**Comment by hattrill** [comment]
GO:1903231 mRNA base-pairing translational repressor activity is not just for miRNAs, but other classes of ncRNA - as it describes the activity not the type of ncRNA, I think it is better than mRNA binding.

The only thing I can think of is to change it to: mRNA base-pairing mediated posttranscriptional gene silencing - which divorces it from translational regulator activity but is a bit of a mouthful.

**Comment by ValWood** [comment]
Ok, we can keep the "translation factors", but in this case we need a clearer separation between "translation factors" which are part of translation and the other types of "translation regulators" that act upstream of translation.
In this case, maybe the simplest solution is to have:


translation regulator
- group all regulators under here
translation factor
- translation initiation factor
- translation elongation factor
- translation termination factor

?

**Comment by hattrill** [comment]
I think that seems like a good idea. I would even suggest that if we can separate the translation factors from this, then 'translation regulator activity' could go as this is vague. But perhaps hive off the translation factors first and see what annotations we are left with.

**Comment by ValWood** [comment]
I will do this.

I did want to annotate this mitochondrial complex as a pre-initiation  "translation activator" as it's a regulator but upsteam of initiation, and it is a specific factor not a general one. That's what began this ticket IIRC.
We'll see..... I'll tackle this next week.

**Comment by ValWood** [comment]
In this case, how would we define "translation factor activity"?

The specific translation factors are defined 
GO:0008079   translation termination factor activity
Functions in the termination of translation.

GO:0008135 translation factor activity, RNA binding
Functions during translation by binding to RNA during polypeptide synthesis at the ribosome.


This would make 
translation factor
Functions during translation 

but this isn't correct because the gene specific regulators upstream of the initiation factors also function during translation.
Any suggestions?

**Comment by pgaudet** [comment]
A molecular function required for translation of a mRNA into a protein. Functions either during initiation, elongation or termination of translation.

**Comment by ValWood** [comment]
I just spotted that @raymond91105  had a comment about regulators.  just as the call ended. Could you paste it here? thanks
Val

**Comment by ValWood** [comment]
We should then tease out the individual activities  for the "factors"
 https://www.nature.com/articles/s41580-023-00624-9?fromPaywallRec=false
and add child terms with parents to actual activities

**Comment by raymond91125** [comment]
> I just spotted that @raymond91105 had a comment about regulators. just as the call ended. Could you paste it here? thanks Val

Are ALL "translation factors" involved in ALL translations thus any factors that are only involved in SOME translations are regarded as "translation regulators"?

**Comment by ValWood** [comment]
Good question. I think this is how transcription is modelled (more or less)
All the core machinery is part_of , and the gene specific parts are regulators.
@pgaudet ?

**Comment by pgaudet** [comment]
I dont know the exact biology - but I dont think ALL "translation factors" need to be involved in ALL translations; we can have processes that are mediated by various versions of a complex in different conditions 

@raymond91125 did I understand your question correctly?

**Comment by raymond91125** [comment]
I think in that sense, the various versions whould still be of one MF which participants in ALL translations.
Formally, translation has_part some translation factor activity; and it's not true that translation has_part some translation regulator activity.
Nothing in biology is 100% so ALL should probably be MOST. Then the formal definition (has_part) does not apply.
Just to emphasize that I am no expert in translation terminology.

**Comment by sjm41** [comment]
> Proposal: translation factor def: A molecular function required for translation of a mRNA into a protein. Functions either during part of initiation, elongation or termination of translation.

There are 'recycling/reinitiation factors' - I'm not sure if these also need to be considered here.
We wrote a short review of the fly cyto & mito translation factors a few years ago  that might be helpful in terms of general classification - contains citations to several more comprehensive reviews:
https://www.tandfonline.com/doi/full/10.1080/19336934.2016.1220464#abstract

**Comment by ValWood** [comment]
Not all initation factors are involved in all iniitations. SOme are only involved in initiation in response to stress.
The distinction  we are trying to make for the regulators is that they act pre-initiation.

The current definition of initiation is "
Definition | Parents
The process preceding formation of the peptide bond between the first two amino acids of a protein. This includes the formation of a complex of the ribosome, mRNA or circRNA, and an initiation complex that contains the first aminoacyl-tRNA. 

The important part of this definition is really the second clause, if my understanding is correct, initiation is the formation of the formation of a complex of the ribosome, mRNA or circRNA, and an initiation complex that contains the first aminoacyl-tRNA. 

the steps which support individual RNAs incorporation into the initiation complex are called 'translation regulators' but not called translation factors . I guess this is because they are not part of the translation machinery per se (perhaps this should be in the definitions of the translation  and translation factor terms?).

**Comment by ValWood** [comment]
>There are also 'recycling/reinitiation factors' - I'm not sure if these also need to be considered here.

good point. 
most of the "recycling factors appear to also be described as initiation and termination factors. For example
translation release factor activity mrf1 is a recycling factor

we also have a term "translation reinitiation," which is part of initiation (reinitiation appear to be  considered to be part of ribosome recycling). We  have a term "GO:0032790  ribosome disassembly" which is not related to translation (but has the narrow synonym ribosome recycling. 

It has one child "rescue of stalled ribosome" (which is linked to translational elongation)

I'm hoping that all of the recycling factors are also initiation or termination factors and this part can be punted. 

I'm not sure that "ribosome disassembly" is a great term, because the ribosome is dissembled by different pathways in different contexts
i) ribosome quality control pathways (to deal with problems during elongation)
ii) ribosome recycling (after termination)

but we might be able to deal with this later

**Comment by ValWood** [comment]
I wonder if, because we know all of the translation factors (I can't remember a new cytosolic one being published while working on fission yeast?),  I can just list the complexes in the respective definition gloss. This shouldn't really be an annotation issue  because everyone knows which gene products/complexes are tifs-  it should only be an ontology issue.
Once the rearrangement is done it will be easy to use the matrix to check for annotation outliers.

**Comment by ValWood** [comment]
I won't to move forward with this so that I can model mitochondrial translation. 
I therefore added a "translation factor activity' grouping term, and moved the current terms underneath it, while only removing the relationships to translation regulator. 
I think some of this is still incorrect with respect to whether or not the translation factors are RNA binding or not, but that can be refined later once the overall structure is correct.

the new translation factor term has the following children:

<img width="456" alt="Image" src="https://github.com/user-attachments/assets/cdce14e5-a56b-4b23-8def-a6d5fb0789f4" />

This also allows me to deal with the children of
GO:0090079 translation regulator activity, nucleic acid binding 
so that it can be obsoleted

**Comment by ValWood** [comment]
It turned out "translation factor activity" was an exact synonym of "translation regulator" activity
so I have removed that. But after the split between translation factors and translation regulators is live we will need to review annotation (I can do this with the Matrix easily)

**Comment by ValWood** [comment]
To make matters worse

Name
translation regulator activity
Definition
Any molecular function involved in the initiation, activation, perpetuation, repression or termination of polypeptide synthesis at the ribosome. Source: GOC:ai

So  translation regulator has a definition for translation factor.
I will suggest a revised definition, but should this really be obsoleted and reinstated?

@pgaudet

**Comment by pgaudet** [comment]
If the annotations will end up being incorrect, then we need to obsolete. Unless it's a manageable number and we can fix them. 

Have you done a check  for annotation usage?

**Comment by ValWood** [comment]
There are only 26 direct EXP annotations

ZFIN:ZDB-GENE-030131-8656	ribosomal protein L10	GO:0045182	UniProt	NCBITaxon:7955	protein_coding_gene	PMID:25316788|ZFIN:ZDB-PUB-141016-1
SGD:S000002758	Specific translational activator for the mitochondrial ATP6 mRNA	GO:0045182	SGD	NCBITaxon:559292	protein	PMID:17110482
SGD:S000002758	Specific translational activator for the mitochondrial ATP6 mRNA	GO:0045182	SGD	NCBITaxon:559292	protein	PMID:17110482
SGD:S000002605	Mitochondrial translational activator of the COB mRNA	GO:0045182	SGD	NCBITaxon:559292	protein	PMID:3329045
UniProtKB:P62395	Secretion monitor	GO:0045182	EcoCyc	NCBITaxon:83333	protein	PMID:15302932
FB:FBgn0028734	Fragile X messenger ribonucleoprotein 1	GO:0045182	CACAO	NCBITaxon:7227	protein	FB:FBrf0141416|PMID:11733059
TriTrypDB:LmjF.35.3980	4E-interacting protein, putative	GO:0045182	GeneDB	NCBITaxon:347515	mRNA	PMID:21764780
FB:FBgn0000392	cup	GO:0045182	UniProt	NCBITaxon:7227	protein	FB:FBrf0167488|PMID:14723848
UniProtKB:P46781	Small ribosomal subunit protein uS4	GO:0045182	UniProt	NCBITaxon:9606	protein	PMID:18420587
UniProtKB:P27635	Large ribosomal subunit protein uL16	GO:0045182	UniProt	NCBITaxon:9606	protein	PMID:26290468
UniProtKB:P62263	Small ribosomal subunit protein uS11	GO:0045182	UniProt	NCBITaxon:9606	protein	PMID:3683397
UniProtKB:C0HM02	PRKCH upstream open reading frame 2	GO:0045182	UniProt	NCBITaxon:9606	protein	PMID:19797084
UniProtKB:C0HM02	PRKCH upstream open reading frame 2	GO:0045182	UniProt	NCBITaxon:9606	protein	PMID:34593629
UniProtKB:Q9NZI8	Insulin-like growth factor 2 mRNA-binding protein 1	GO:0045182	BHF-UCL	NCBITaxon:9606	protein	PMID:9891060
SGD:S000005199	Translational activator for mRNAs with internal ribosome entry sites	GO:0045182	SGD	NCBITaxon:559292	protein	PMID:21277287
SGD:S000004870	Mitochondrial translational activator specific for the COX2 mRNA	GO:0045182	SGD	NCBITaxon:559292	protein	PMID:3034718
SGD:S000005328	Mitochondrial translational activator specific for the COX3 mRNA	GO:0045182	SGD	NCBITaxon:559292	protein	PMID:2833749
SGD:S000004668	Protein required for expression of the mitochondrial OLI1 gene	GO:0045182	SGD	NCBITaxon:559292	protein	PMID:21454598
PlasmoDB:PF3D7_1200610	VAR2CSA uORF	GO:0045182	GeneDB	NCBITaxon:36329	mRNA	PMID:19119419
MGI:MGI:1096372	Y-box-binding protein 2	GO:0045182	MGI	NCBITaxon:10090	protein	PMID:12297523
SGD:S000005828	CPA1 uORF	GO:0045182	SGD	NCBITaxon:559292	protein	PMID:10608810
SGD:S000002227	Mitochondrial translational activator of the COB mRNA	GO:0045182	SGD	NCBITaxon:559292	protein	PMID:3033440
AGI_LocusCode:AT5G21160	AT5G21160	GO:0045182	TAIR	NCBITaxon:3702	protein	TAIR:Publication:501803459|PMID:33054972
SGD:S000003454	Mitochondrial inner membrane protein	GO:0045182	SGD	NCBITaxon:559292	protein	PMID:2548921
SGD:S000004193	Specific translational activator for the mitochondrial COX1 mRNA	GO:0045182	SGD	NCBITaxon:559292	protein	PMID:19710419
SGD:S000004057	Specific translational activator for the COX1 mRNA	GO:0045182	SGD	NCBITaxon:559292	protein	PMID:7664742


Not so many. But they don't all look correct.

It might be cleaner to create a new term, move the descendants to it, and obsolete this term.

I will review these before deciding

**Comment by ValWood** [comment]
From a review, there are no annotations to "translation factors", only to "translation regulators" or incorrect annotations.

Perhaps the best approach is 
- [ ] keep the term, and correct the definition (which reflects the usage)
and
- [ ] dispute the incorrect annotations.

**Comment by ValWood** [comment]
The new organization has correctly classified the PomBase translation factors/ regulators

<img width="1063" alt="Image" src="https://github.com/user-attachments/assets/955a176c-cfdf-4c25-a905-eac1a947b0f8" />

translation factor https://www.pombase.org/results/from/id/f4dfe0a0-3b2f-4d04-a3b9-5ec96fe5fc74

translation regulator  https://www.pombase.org/results/from/id/3002affd-fa1d-4333-abaa-721ff283ed13

Once AmiGO is updated, we can use the matrix to check that the intersection is zero globally

**Comment by ValWood** [comment]
This is closer.

Translation regulators  are largely OK (with one incorrect relation, see below)

<img width="599" alt="Image" src="https://github.com/user-attachments/assets/38b76df7-080d-4c40-81f6-f52672d582f5" />

Translation factors

<img width="822" alt="Image" src="https://github.com/user-attachments/assets/fae88b50-265d-47fe-855d-205ac866f079" />

Still to do

- [ ] remove the link between translation repressor, non nucleic acid binding
and translation factor activity, non-nucleic acid binding
- [ ] translation factor activity RNA binding is still a parent of the translation factors. Need to deal with this based on annotations

**Comment by ValWood** [comment]
- [ ]. The definition of translation regulator is too braod
- translation regulator activity
Any molecular function involved in the initiation, activation, perpetuation, repression or termination of polypeptide synthesis at the ribosome. Source: GOC:ai

I will change this to 
Any molecular function involved **in the regulation of initiation**, activation, perpetuation, repression or termination of polypeptide synthesis at the ribosome.

**Comment by ValWood** [comment]
- [x]  since translation factor activity is a new term, I added gocheck_do_not _annotate, should always know the subclass

**Comment by ValWood** [comment]
Just spotted 
translation factor activity, non-nucleic acid binding
will check if this can also be obsoleted

**Comment by ValWood** [comment]
GO:0045183 translation factor activity, non-nucleic acid binding (2 EXP).

l would suggest obsolete with direct replacement  "translation factor activity"
but I made "translation factor activity" and "do not annotate", and really, these would be better moved to the appropriate child factor, i.e. initiation, elongation, termination.

However, I don't think either of these 2 are "translation factors"?

MCTS1 | Malignant T-cell-amplified sequence 1 Mouse, UniProt
Neurl1 | E3 ubiquitin-protein ligase NEURL1 Human Uniprot
will check /dispute before I open the obsoletion ticket.

@pgaudet are you happy for me to propose obsoletion of
GO:0045183 translation factor activity, non-nucleic acid binding (2 EXP).
and
GO:0008135 translation factor activity, RNA binding (9 EXP)

**Comment by ValWood** [comment]
Neurl1 | E3 ubiquitin-protein ligase NEURL1 Human Uniprot
is a ligase part of regulation of translation (not really a translation regulator, and def not a translation factor)

Involved in the formation of spines and functional synaptic contacts by modulating the translational activity of the cytoplasmic polyadenylation element-binding protein CPEB3. Promotes ubiquitination of CPEB3, and hence induces CPEB3-dependent mRNA translation activation of glutamate receptor GRIA1 and GRIA2. Can function as an E3 ubiquitin-protein ligase to activate monoubiquitination of JAG1 (in vitro), thereby regulating the Notch pathway. Acts as a tumor suppressor; inhibits malignant cell transformation of medulloblastoma (MB) cells by inhibiting the Notch signaling


MCTS1 | Malignant T-cell-amplified sequence 1 Mouse, UniProt
seems to be a translation initiation factor involved in reinitiation/recycling (it also has a KW mapping to translation initiation factor)
